shared_model/validators/field_validator.cpp

```diff
@@ -403,6 +403,42 @@ namespace shared_model {
       return boost::none;
     }
 
+    boost::optional<ValidationError> validatePaginationOrdering(
+      interface::TxPaginationMeta::Ordering const& ordering) {
+      using Field     = interface::TxPaginationMeta::Field;
+      using Direction = interface::TxPaginationMeta::Direction;
+
+      for (auto const& ordering_entry : ordering) {
+        auto const ord_field     = ordering_entry.first;
+        auto const ord_direction = ordering_entry.second;
+
+        if (ord_field < Field::field_first || ord_field > Field::field_last) {
+          return ValidationError("Ordering",
+            {
+              fmt::format("Passed field value is {} , while it must be between {} and {} "
```

Comment:
this error message is what API user gets. we should not include iroha internal magic constants here I would just write "Unknown field."